Bottom: 0ec0a3522e435dece2e4abd709be380e722ad2a0
Top:    32e9cfd6aa8fc689f1837d0276c689dc09e8e6dd
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-31 16:30:19 -0500

Refresh of buildbot-worker-ptxdist

---
diff --git a/docker/buildbot-worker-ptxdist/buildbot/start.sh b/docker/buildbot-worker-ptxdist/buildbot/start.sh
index 23ce5835..a6343a88 100755
--- a/docker/buildbot-worker-ptxdist/buildbot/start.sh
+++ b/docker/buildbot-worker-ptxdist/buildbot/start.sh
@@ -1,8 +1,12 @@
 #!/bin/sh
 
-mkdir -p $HOME/.ssh
-cp -R /run/secrets/ssh/* $HOME/.ssh
-chmod -R 600 $HOME/.ssh/*
+SSH_BITS=${SSH_BITS-"2048"}
+SSH_TYPE=${SSH_TYPE-"rsa"}
+SSH_FILE=${SSH_FILE-"$HOME/.ssh/id_rsa"}
 
-ln -v -s -f /home/buildbot/buildbot.tac /buildbot/buildbot.tac
+BUILDBOT_DATA=${BUILDBOT_DATA-"/buildbot"}
+
+ssh-keygen -q -b ${SSH_BITS} -t ${SSH_TYPE} -N '' -f ${SSH_FILE} 0>&- 1>/dev/null 2>/dev/null
+
+ln -v -s -f ${HOME}/buildbot.tac $BUILDBOT_DATA
 exec twistd --nodaemon --logfile=- --pidfile=/tmp/twistd.pid --python=buildbot.tac
